Sources/LogView/LogTableViewController.swift

```diff
@@ -34,6 +34,18 @@ class LogTableViewController: UIViewController {
     var selectedLogItem: LogItem!
     var domainSections: [CollapsibleSection<LogItem>] = []
     
+    override var preferredStatusBarStyle: UIStatusBarStyle {
```

Comment:
Bad merge conflict? I think all these added lines should have been removed

Reply:
I believe so yeah, will remove!